engine-tests/src/test/java/org/terasology/utilities/NoiseTest.java

```diff
@@ -51,11 +51,11 @@ public NoiseTest(Noise noiseGen) {
     @Parameters(name = "{0}")
     public static Collection<Noise[]> data() {
         return Arrays.asList(new Noise[][]{
-                { new WhiteNoise(0xCAFE) },
-                { new DiscreteWhiteNoise(0xCAFE) },
-                { new SimplexNoise(0xCAFE) },
-                { new PerlinNoise(0xCAFE) },
-                { new BrownianNoise(new WhiteNoise(0xCAFE), 3) }
```

Comment:
@msteiger: to be clear, are you suggesting LadySerenaKitty to revert this change and modify the checkstyle rule right in this PR?